frontend/packages/topology/src/components/page/TopologyView.tsx

```diff
@@ -320,25 +322,29 @@ export const ConnectedTopologyView: React.FC<ComponentProps> = ({
     }
   }, [searchParams, labelParams]);
 
+  const nodesLength = filteredModel?.nodes?.length;
+
   const viewContent = React.useMemo(
     () =>
-      viewType === TopologyViewType.list ? (
-        <TopologyListView
-          model={filteredModel}
-          namespace={namespace}
-          onSelect={onSelect}
-          setVisualization={setVisualization}
-        />
-      ) : (
-        <Topology
-          model={filteredModel}
-          namespace={namespace}
-          application={applicationRef.current}
-          onSelect={onSelect}
-          setVisualization={setVisualization}
-        />
-      ),
-    [filteredModel, namespace, onSelect, viewType],
+      nodesLength && (nodesLength < 100 || (nodesLength > 99 && showTopologyAnyway)) ? (
```

Comment:
```suggestion
      nodesLength < 100 || showTopologyAnyway ? (
```

Use a const for the max nodes.

Also, do we really want to render `null` if 0 nodes? old code didn't